libarchive/archive_read_disk_entry_from_file.c

```diff
@@ -103,6 +103,8 @@ __FBSDID("$FreeBSD");
 
 static int setup_mac_metadata(struct archive_read_disk *,
     struct archive_entry *, int *fd);
+static int setup_xattrs_namespace(struct archive_read_disk *,
+    struct archive_entry *, int *, int);
```

Comment:
This needs to be enclosed in `#ifdef ARCHIVE_XATTR_FREEBSD` or we get unused function errors on other platforms

Reply:
Good catch. Will do. Thanks!